### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by yaroslavGoncharuk**
- updated b2b branch
 - fixed composer.json copying for b2b edition
 - fixed redis service restart
 - added mail service

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,7 +1,7 @@
 COMPOSE_PROJECT_NAME=storefront
 MAGENTO_PATH=~/projects/magento/storefront/repos
 MAGENTO_DOMAIN=magento.test
-INSTALLED_REPOS="magento2ce magento2ee saas-export catalog-storefront catalog-storefront-ee"
+INSTALLED_REPOS="magento2ce magento2ee magento2b2b saas-export catalog-storefront catalog-storefront-ee"
```
Comment (mslabko) [inline .env]: By default we don’t want to install b2b
And ee as well in future

File: bundles/typical.yml [hunk 2]
```diff
@@ -134,6 +134,20 @@ services:
     x-setup:
       - bin/magento setup:config:set -n --amqp-host=rabbit --amqp-user=guest --amqp-password=guest
     x-info: https://devdocs.magento.com/guides/v2.3/config-guide/mq/manage-message-queues.html
+#  mail:
```
Comment (mslabko) [inline bundles/typical.yml]: only storefront.xml should be updated

**Review by mslabko — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (yaroslavGoncharuk) [reply .env]: fixed

File: bundles/typical.yml (hunk 2 above)
Comment (yaroslavGoncharuk) [reply bundles/typical.yml]: fixed

**Review by mslabko — APPROVED** [review]

**Pull request merged**